ui/test/nuxeo-favorites-toggle-button.test.js

```diff
@@ -48,12 +49,19 @@ suite('nuxeo-favorites-toggle-button', () => {
       ]);
     });
 
+    teardown(() => {
+      if (window.confirm.restore) {
+        window.confirm.restore();
+      }
+    });
+
     test('it should display the document as favorite', () => {
       expect(element.favorite).to.be.true;
     });
 
-    test('toggle should remove document from favorites', async () => {
+    test('toggle should remove document from favorites when clicked ok', async () => {
```

Comment:
We still need to test that it should not remove the document from favorites if the user clicks cancel.